doc/source/user/basics.ufuncs.rst

```diff
@@ -82,7 +82,8 @@ an integer (or Boolean) data-type and smaller than the size of the
 Finally, the *out* keyword allows you to
 provide an output array (for single-output ufuncs, which are currently the only
 ones supported; for future extension, however, a tuple with a single argument
-can be passed in). If *out* is given, the *dtype* argument is ignored.
+can be passed in). If *out* is given, the *dtype* argument is only used for the
+internal computations.
```

Comment:
```suggestion
can be passed in). If *out* is given, the *dtype* argument is ignored.
```
Reverting this change since https://github.com/numpy/numpy/pull/25431/files#r1451835427 is unresolved. Please open another PR if you wish to add details here, and ask for someone familiar with ufunc implementation to review.